TestAssets/DesktopTestProjects/AutoAddDesktopReferencesDuringMigrate/project.json

```diff
@@ -4,7 +4,9 @@
     "debugType": "portable",
     "emitEntryPoint": true
   },
-  "dependencies": {},
```

Comment:
have filed an issue for removing this : https://github.com/dotnet/cli/issues/5236